client/src/components/SupportTools/SupportTools.test.tsx

```diff
@@ -175,20 +175,9 @@ const apiCalls = {
   },
 }
 
-const queryClientOptions = queryClient.getDefaultOptions()
-
 const renderRoute = (route: string) =>
   renderWithRouter(
-    <QueryClientProvider
-      client={
-        new QueryClient({
-          defaultOptions: {
-            ...queryClientOptions,
-            queries: { ...queryClientOptions.queries, retry: false },
-          },
-        })
-      }
-    >
+    <QueryClientProvider client={queryClient}>
```

Comment:
This change is just to use our main query client like we do in all other tests. This test never got updated because it was the first to use react-query